dataland-frontend/src/assets/css/main.css

```diff
@@ -4,4 +4,14 @@
   -moz-osx-font-smoothing: grayscale;
   text-align: center;
   color: var(--text-color);
-}
\ No newline at end of file
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+}
+
+.text-color { color: var(--text-color); }
```

Comment:
what were the acutal changes in theme.css?

Reply:
this line is not needed at the moment i removed it

Reply:
ok